### ISSUE TITLE
merge from upstream: editor supraline fix code

### ISSUE THREAD

**Issue by rla2118** [DCLP/dclpxsltbox#159]
http://dclp.github.io/dclpxsltbox/output/dclp/131/130167.html
The display of overstrokes on the first two characters (ιη) of the first line in this record is not right. The line should be continuous over both iota and eta.

XML doesn't appear to be correct:
It currently has 
`< lb n="1"/><num value="18">ῑ<hi rend="supraline">η</hi></num> δ<unclear>ι</unclear>’ ὅλης ἡ<unclear>μέρας</unclear>.`

But should be, I suppose,
`< lb n="1"/><num value="18"><hi rend="supraline">ιη</hi></num> δ<unclear>ι</unclear>’ ὅλης ἡ<unclear>μέρας</unclear>.`

I'm sending to @jcowey for comment and/or forwarding to someone else.

**Comment by paregorios** [comment]
I've just updated comment from @rla2118 above using three continuous grave accents before and after code snippets so all pointy brackets show up in view

**Comment by paregorios** [comment]
And, FWIW, the encoding recommendation/fix from @rla2118 looks correct to me.

**Comment by jcowey** [comment]
This is a known problem with the editor. 
In Leiden+ if one adds a supraline to the letters alpha or iota it is not written as proper xml mark up. One can change to proper xml mark up in the xml tab and then preview is correct and subly different from when supraline is added via Leiden+. (The overstroke sits higher when correct).
Ryan has a fix for it which has not yet been implemented on papyri.info where it is also a problem. Most people are unaware of this. I personally correct when finalising and then it goes into the system correctly. 
I looked for a ticket in which this is described and where Ryan proposes the fix, but I cannot find it. Sorry I cannot explain why it happens.

**Comment by paregorios** [comment]
@ryanfb can you point us to a ticket (if one exists) as discussed by @jcowey above? I'd prefer to close this ticket with a pointer to wherever it's being tracked across the larger effort. thanks.

**Comment by ryanfb** [comment]
See discusion at sosol/sosol#121 (and linked), and particularly now-passing tests in `rails-3-nfd` branch of [papyri/xsugar](https://github.com/papyri/xsugar) at: https://github.com/papyri/xsugar/compare/rails-3...rails-3-nfd#diff-a4da66e842ac3a281fe18b9ea50a3345R1069

**Comment by jcowey** [comment]
ryanfb has made the fix on production in papyri.info, tested it and it is fine. I have cross checked and confirm that it is working for me on production in papyri.info.
This means that this should now no longer happen. That is great. There probably remain a few instances to be fixed. Once that is done everything should be fine.
Closing this and opened a new issue: #161

**Comment by paregorios** [comment]
Reopening this ticket as we will need to pull changes made by @ryanfb into our fork and branch if we want to see the fix working when we get our version of editor up and running. Taking assignment of this ticket for now ahead of coordination with @Edelweiss to schedule that update/merge.

**Comment by jcowey** [comment]
Blocked until our editor is up and usable.

**Comment by paregorios** [comment]
@jcowey @Edelweiss is this still blocked or should we put it back in motion

**Comment by paregorios** [comment]
Adding this issue to the WBWorkshop milestone since it is a blocker for #161, which is already in that milestone.

**Comment by paregorios** [comment]
I gather that this issue is no longer blocked, since we now have a working editor. I'm going to remove the "blocked" tag, bump the priority (since it's a blocker for another item in the current milestone), and reassign to @Edelweiss

**Comment by Edelweiss** [comment]
The master branch of our xsugar repository is now up-to-date with papyri’s master branch.

https://github.com/DCLP/xsugar/commits/master

Is there anything else we need to take into account (rails-3, rails-3-nfd), or does master comprise all the changes that are necessary for the desired feature to work properly?

**Comment by paregorios** [comment]
I think [that's a question](https://github.com/DCLP/dclpxsltbox/issues/159#issuecomment-281624178) for @ryanfb ...

**Comment by paregorios** [comment]
@ryanfb can you answer [the question @Edelweiss poses above](https://github.com/DCLP/dclpxsltbox/issues/159#issuecomment-281624178)? I'd like to get this ticket resolved.

**Comment by ryanfb** [comment]
@paregorios @Edelweiss Papyri's master branch should have everything we know of for supraline. I believe there may still be some outstanding issues we haven't resolved in any branch:

https://github.com/papyri/xsugar/issues/4
https://github.com/papyri/xsugar/issues/5
https://github.com/papyri/xsugar/issues/10

**Comment by paregorios** [comment]
thanks @ryanfb 

@Edelweiss back to you ... perhaps best to ask @jcowey to confirm that the "it's working" notice he provided above holds true on the current deployment of litpap.info?

**Comment by paregorios** [comment]
ping @jcowey @Edelweiss
